var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -48,6 +48,9 @@ class Curl(AutotoolsPackage):
     variant('libssh',     default=False, description='enable libssh support')  # , when='7.58:')
     variant('darwinssl',  default=sys.platform == 'darwin', description="use Apple's SSL/TLS implementation")
     variant('gssapi',     default=False, description='enable Kerberos support')
+    variant('librtmp',    default=False, description='enable Rtmp support')
+    variant('ldap',       default=False, description='enable ldap support')
+    variant('libidn2',    default=True,  description='enable libidn2 support')
```

Comment:
I know this was previously a required dependency, but should we default this to False to minimize the number of deps?

Reply:
I don't mean either way, what's the guideline . To minimize dependencies in default settings.

Reply:
There isn't a strict guideline at the moment. Right now, Spack builds all packages from source code and there are no public binaries for packages, so I tend to disable most variants, especially the ones that add additional dependencies. Once public binaries become available, this guidance may change and we'll likely default to the most commonly used/useful variants being enabled by default.

Reply:
changed to false